### DISCUSSION TITLE
Share your results and the used settings.

### DISCUSSION THREAD

**Discussion by sb53systems** [sb53systems/G-Code-Flow-Temperature-Controller#8] [Show and tell]
[ARTICULATED poison DRAGON flexible](https://www.thingiverse.com/thing:6428358)  
Printed at 60% Scale  
![346751554-a0fea6c2-5319-4f6e-85f8-fee43d5475e9](https://github.com/user-attachments/assets/663033d6-94d1-400c-a19d-1a7f98492ce5)  
![346751938-35a6d8fb-c97c-464f-9ef0-80b3a0a37ea8](https://github.com/user-attachments/assets/e45ec26f-b143-4f2a-95d9-7ccce2c40688)  
![346763612-1342e123-4b55-48cc-9317-1603665bffec](https://github.com/user-attachments/assets/e69c5953-63ba-452b-932f-c4eb2a13763c)  
![346763447-e3ff58ee-b3bb-4668-876a-79d4eed1132a](https://github.com/user-attachments/assets/93b1a4cf-f25e-4a1a-a89d-541df3677d64)  
![346763721-8ccfaf75-47e4-4ad5-96cd-8f10c3dcfc18](https://github.com/user-attachments/assets/2388562a-69ee-4e31-8c06-b75d57a75dd8)  
![346771507-74637585-1eea-47ea-9310-e4e78d7ae3ba](https://github.com/user-attachments/assets/c0964084-d29d-4e26-8bde-59e789977bfa)

**Comment by sb53systems** [comment]
https://www.thingiverse.com/thing:6031589
![image](https://github.com/user-attachments/assets/7e31e0ec-de98-451b-8582-97559039f1e1)  
![image](https://github.com/user-attachments/assets/51072627-ad8a-4a03-b6c2-7b2fe6c65862)  
![image](https://github.com/user-attachments/assets/895879d7-6b5f-4f48-81bb-440331d5cf95)  
  
![IMG20240907150220](https://github.com/user-attachments/assets/c319b47c-2b57-464a-9db6-8b5ab04ee34a)  
![IMG20240907150312](https://github.com/user-attachments/assets/99184365-268e-4720-a7e8-b12f69dc6a5e)

**Comment by sb53systems** [comment]
![IMG_20240907_213751](https://github.com/user-attachments/assets/c8e0f0d3-4676-48f9-8d4d-c796653ac60b)  
https://www.thingiverse.com/thing:308335  
Printed at 50% Scale.  
Print Time: 1H30.   
  
# Part 1  
![image](https://github.com/user-attachments/assets/e940246e-4a44-419f-b353-82217c04706f)  
![image](https://github.com/user-attachments/assets/5a098e75-f10a-42e4-b0da-4988d5326d73)  
Below the generated G-Code.  
![image](https://github.com/user-attachments/assets/c3fc329f-f9aa-4317-8dc3-8edc360fe9cd)  
  
# Part 2
![image](https://github.com/user-attachments/assets/93d0310a-4a08-48f7-827f-1e68ebdef65e)  
![image](https://github.com/user-attachments/assets/9436b539-b879-42d5-b2f2-56147d0ebc14)  
Below the generated G-Code.  
![image](https://github.com/user-attachments/assets/7bb435ed-e5c5-46fb-9d5e-28b1c830d039)  
  
![IMG_20240907_212537](https://github.com/user-attachments/assets/07dadf34-6db5-4cc2-bfca-ad831a534b9c)  
![IMG_20240907_212604](https://github.com/user-attachments/assets/1f906504-1381-4a58-b02a-bfcfe97db519)  
![IMG_20240907_212505](https://github.com/user-attachments/assets/4c0a8c0d-a558-4c6d-bbcb-bf3f9b0d0403)  

![IMG_20240908_003150.jpg](https://github.com/user-attachments/assets/6a73f555-e0d8-4380-bd7a-341e68833916)

**Comment by sb53systems** [comment]
# Voron Cube (Orca Slicer Handy Modal)
![image](https://github.com/user-attachments/assets/ce7987b2-c693-48cc-9662-065e38bf082e)  
![image](https://github.com/user-attachments/assets/30247885-426d-4351-9a0b-ebd718ee8eba)  
  
https://github.com/user-attachments/assets/826f9fc6-22d9-43aa-af12-ef670d3eedfa  
![IMG20240911225759](https://github.com/user-attachments/assets/17b707bb-2cdf-4232-bd47-999d7bd404bb)  
![IMG20240911230229](https://github.com/user-attachments/assets/f28be56e-1c27-40e0-89e9-afd7cf25ebe6)  
![IMG20240911231602](https://github.com/user-attachments/assets/f46c8e4f-6e05-4a5a-912d-ea374d85a7ed)  
  
https://github.com/user-attachments/assets/bcb69a7e-d312-489c-8e26-1e5d2e9df23d

**Comment by sb53systems** [comment]
# A 10 minutes Goblin Dungeon Janitor
https://cults3d.com/en/3d-model/game/goblin-dungeon-janitor  
  
![image](https://github.com/user-attachments/assets/1f7b031a-838d-4f61-be47-74a1cc4e64fc)  
![image](https://github.com/user-attachments/assets/6088ddd7-4091-46be-922d-c10c30f030ec)  
![image](https://github.com/user-attachments/assets/8daee284-40c5-49aa-bf9f-9ef8a5cdb118)  
  
Real-time printing video on youtube: https://www.youtube.com/watch?v=zoNY-nFGmWk&t=9s  
  
![IMG_20240926_004357](https://github.com/user-attachments/assets/b3bdea62-3284-4c17-bb4e-022f23857f41)  
![IMG_20240926_004321](https://github.com/user-attachments/assets/e8898223-8375-402f-8743-63b8ec2ca3b9)  
![IMG_20240926_004538](https://github.com/user-attachments/assets/42198bd6-6298-4ec4-848b-9c0f7473636d)  
![IMG_20240926_010343](https://github.com/user-attachments/assets/c8827eac-04ab-4e29-9d03-d79be84cee24)

**Comment by sb53systems** [comment]
# Printing a 3D Benchy at 30% scale, with 70mm/s speed !
![image](https://github.com/user-attachments/assets/baa941f1-71a7-4cb0-bf56-f26ec5911838)  
![image](https://github.com/user-attachments/assets/5c1e529f-6f92-4f2b-b3d7-d5e5a4ed96d2)  
![image](https://github.com/user-attachments/assets/072e1166-f86d-458b-9bce-2d9fbdb1cad4)  
![371859967-d2910e98-8a20-41a1-9eab-a3c8e1a1a73c](https://github.com/user-attachments/assets/c421492e-fce7-4b0a-9cd4-89667f146207)  
  
Real-time printing video on youtube: https://www.youtube.com/watch?v=ABg8R-nOLmg  
  
![IMG20240929160030](https://github.com/user-attachments/assets/2472047b-a368-4b6e-a08e-98a4df17eba2)  
![IMG20240929153352](https://github.com/user-attachments/assets/9c9a77bc-80d9-43ea-b8a0-074e4ddd1409)  
![IMG20240929153329](https://github.com/user-attachments/assets/c6d9a5e4-a91c-4136-a1d5-91269b0e7f21)  
![IMG_20240929_181654](https://github.com/user-attachments/assets/cf9e1f3b-e132-4688-b719-bb5e73c05e75)

**Comment by sb53systems** [comment]
# A 2m43s 3D Benchy at 20% Scale
![image](https://github.com/user-attachments/assets/df394364-6a4c-442b-ab4a-3333db4ecdd9)  
![image](https://github.com/user-attachments/assets/a3ae5747-2553-4bea-b54b-0716f39fdab9)  
![image](https://github.com/user-attachments/assets/18fcdb2e-fa25-4ef9-ad8c-4b79e2513f20)  
![image](https://github.com/user-attachments/assets/2f5b9157-7cce-4b41-ad8a-b95d9e13d4f4)  
  
Real-time printing video on youtube: https://www.youtube.com/watch?v=9NtXSy6qLak  
  
![IMG20241009002745](https://github.com/user-attachments/assets/67a8e032-bcfb-4fce-9709-317ff9e5f652)  
![IMG20241009002556](https://github.com/user-attachments/assets/aeb49d98-506a-46e3-8ced-93baee0c22b2)  
![IMG20241009003335](https://github.com/user-attachments/assets/af21dd49-a3a8-41f6-af2d-d4f7e9c0514b)

**Comment by sb53systems** [comment]
# [NObody - The Thinker](https://makerworld.com/fr/models/534040)
- Filament: PLA  
- Scale: 50%  
- Support: Tree Auto  
- Seam position: Back  
  
![Capture d’écran 2024-12-11 123158](https://github.com/user-attachments/assets/e440507b-038c-4f41-9bb5-219491e85395)  
![Capture d’écran 2024-12-11 123232](https://github.com/user-attachments/assets/1103efda-4706-4a24-a300-1230beb3b3ec)  
![Capture d’écran 2024-12-11 123351](https://github.com/user-attachments/assets/9013636c-7927-4cf1-941a-1efa0e302195)  
![3de39e4d-e274-46f7-9ce7-e9d36548f9e8](https://github.com/user-attachments/assets/acaaea9d-9502-46ea-97ec-fec2eb7d2670)  
![0f7ee88d-738a-4eeb-a7e3-199b03799fbc](https://github.com/user-attachments/assets/259e76d1-8a1d-4abe-9834-084aab3c4638)  
![912cf85a-3729-4b0a-ad14-5323674bbf02](https://github.com/user-attachments/assets/ed7ffcc1-0e21-4361-812b-271acae81557)

**Comment by sb53systems** [comment]
# [A pair of Christmas reindeer sculpture ornaments](https://makerworld.com/en/models/863149)
![image](https://github.com/user-attachments/assets/a6a08f1a-332e-4757-984a-2aa4015316f1)  
![image](https://github.com/user-attachments/assets/058ec53d-11f9-4805-9566-421c97026799)  
![image](https://github.com/user-attachments/assets/d2f263a0-6d3b-47f8-b6a8-8fc6435c6565)  
![4465c3f8-6d4f-408b-bd05-dbb59096c96f](https://github.com/user-attachments/assets/203c40b6-9bfe-4fce-a5e7-1246c60b6bf5)  
![0c8942a8-eda5-4000-b2d4-e6ce75dce7a6](https://github.com/user-attachments/assets/d82a464f-34e7-48f0-8234-73c5d7fd7abe)  
![bca5b34e-72ab-4591-8d58-592d18993f77](https://github.com/user-attachments/assets/56556f4d-5452-4d5b-8a0f-64b8c6d8ae80)  
![f17dbeca-8dfd-4ddb-9023-a040d2dddf4a](https://github.com/user-attachments/assets/c1604805-3d02-46de-87bf-dbaae300654c)

**Comment by GromovPeter** [comment]
Hi
great job!
use it always
Need to support ironing )

**Comment by valerii-bokhan** [comment]
Printed with 0.02mm layer height, 12.5 hours on Bambu Lab A1 mini.

Settings and info were taken from this post: https://www.reddit.com/r/FDMminiatures/comments/1my1r2h/printing_fdm_miniatures_with_a_postprocessing/

![photo_5294346640808867873_y](https://github.com/user-attachments/assets/c014f3ff-36f6-4da2-84df-94972d5e0e2f)
![photo_5294346640808867872_y](https://github.com/user-attachments/assets/a3ba1e5a-4628-4f71-a492-c656d08ae348)
![photo_5294346640808867871_y](https://github.com/user-attachments/assets/6c023e96-6714-4b0a-a60f-522b8408fb7b)
![photo_5294346640808867870_y](https://github.com/user-attachments/assets/3f2e72fe-05d5-4211-8614-48f0960c3153)
![photo_5294346640808867869_y](https://github.com/user-attachments/assets/bdce3e8a-d8dc-4eb6-978f-b4e033369f1f)
![photo_5294346640808867868_y](https://github.com/user-attachments/assets/a9fe9e4c-3670-4a36-ba06-c55997118752)
![photo_5294346640808867866_y](https://github.com/user-attachments/assets/b68cc1fe-f00a-4dae-9209-75eaf771b79c)
![photo_5294346640808867867_y](https://github.com/user-attachments/assets/5f3b2aed-af2e-4efa-8a6b-205a752e2485)
